src/test/java/seedu/toluist/storage/JsonStorageTest.java

```diff
@@ -47,46 +54,116 @@ public void setUp() {
 
     @Test
     public void load_emptyData() {
-        Optional<TodoList> todoListOptional = loadDataFromPath(TEST_STORAGE_FILE_PATH + "EmptyData.json");
-        assertTrue(todoListOptional.isPresent());
-        assertEquals(todoListOptional.get(), new TodoList());
+        TodoList todoList = null;
+        try {
+            todoList = loadDataFromPath(EMPTY_JSON_DATA_FILE_PATH);
+        } catch (IOException e) {
+            fail("Should not throw exception");
+        }
+        assertEquals(todoList, new TodoList());
     }
 
     @Test
     public void load_invalidJson() {
-        Optional<TodoList> todoListOptional = loadDataFromPath(TEST_STORAGE_FILE_PATH + "NotJsonFormatData.json");
-        assertFalse(todoListOptional.isPresent());
+        try {
+            loadDataFromPath(INVALID_JSON_DATA_FILE_PATH);
```

Comment:
add a fail at this line so if exception doesnt get thrown it will fail this test?